UI/message.ts

```diff
@@ -41,7 +42,13 @@ function* match(regex: RegExp, content: string, type: ItemType): Generator<Conte
                 };
             }
         } else if (type == "npub") {
-            const pubkey = PublicKey.FromBech32(content.slice(urlStartPosition, urlEndPosition + 1));
+            let bech32: string;
+            if (match[0].startsWith("nostr:")) {
+                bech32 = content.slice(urlStartPosition + 6, urlEndPosition + 1);
+            } else {
+                bech32 = content.slice(urlStartPosition, urlEndPosition + 1);
+            }
+            const pubkey = PublicKey.FromBech32(bech32);
```

Comment:
Actually, if you think about it, it's a better idea to let `FromBeck32` accept both formats. Not a PR blocker, but what's your opinion?

Reply:
I have thought about this, put the if logic above into the PublicKey API but then i feel, when i read FromBech32, I wont think about accepting nostr: format. Only for readableness. I will use PublickKey.FromNostrURL such thing, and it accept both nostr:naddr and nostr:nprofile, PublickKey.FromBech32 will accept naddr and nprofile